task-48748: pass user enabled/disabled/deleted status to extension registry components

hthabti opened the pull request:
Prior to this change it was possible to perform actions with deleted/disabled users
fix:
 - We will pass the user status enabled/disabled/deleted to the other components injected with the extension registry to allow components to set specific behavior depending on the user status.

halaya committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

halaya committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

halaya committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

hthabti committed:
task-48748: disable kudos for disabled users

prior to this change it was possible to try to send kudos to disabled users
fix:

update on buttonDisabled to check if receiver's identity is enabled so if its value is false the action button on comments and activities would be disabled
display/not display kudos icon for users popover

boubaker commented:
we don't have kudos in social project, please change the title to mention what behavior you fixed in social.

ahamdi commented on webapp/portlet/src/main/webapp/vue-apps/common/components/ExoUserAvatar.vue:
```diff
@@ -267,12 +270,15 @@ export default {
       return {
```
Where do we use this computed property ?

ahamdi replied:
Could you update the title to reflect the change ? 
Suggestion : 
Pass user enabled/disabled status to extension registry components

hthabti committed:
Pass user enabled/disabled/deleted status to extension registry components

wassimmhamdi committed:
Task-54820: unify the Title Style of the External spaces list portlet (#1623)

Problem: The external spaces portlet title style is different than the style of the other widgets in the snapshot page
Fix: Unify the title style of external spaces portlet to be coherent with the other portlet's styles

hthabti committed:
Merge branch 'feature/maintenance' into task-48748

hthabti committed:
Merge branch 'feature/maintenance' into task-48748

ahamdi committed:
Merge branch 'feature/maintenance' into task-48748

ahamdi committed:
Merge branch 'feature/maintenance' into task-48748
